market/connector/adobe-esign-connector/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "adobe-esign-connector",
+    "name": "Adobe Esign Connector",
+    "description": "Use this connector to create, sign document with Adobe Sign.",
+    "type": "connector",
+    "platformReview": "4.5",
+    "sourceUrl": "https://github.com/axonivy-market/adobe-esign-connector",
+    "statusBadgeUrl": "https://github.com/axonivy-market/adobe-esign-connector/actions/workflows/ci.yml/badge.svg",
+    "language": "English",
+    "industry": "Cross-Industry",
+    "tags": [ "e-signature" ],
+    "mavenArtifacts": [
+        {
+            "repoUrl": "https://maven.axonivy.com",
+            "name": "Adobe Esign Connector",
```

Comment:
Do you need to change all these names to the official name, see line 4 above. I dont like to have different names afterwards and "Adobe Acrobat Sign" is supposed to be the right one.

Reply:
I am puristic so I would say yes... but I am not aware if this causes additional problems we dont want. Maybe @ivy-sgi, you can share your opinion here?